app/assets/stylesheets/application.scss

```diff
@@ -8,8 +8,12 @@
 
 @import "reset";
 
+// some colours used below
+$manual-search-button-colour: #0b0c0c;
```

Comment:
I think this can be set to `$black`:
https://github.com/alphagov/govuk_frontend_toolkit/blob/89168934d1455754f455c398ba5c350a484d9a2a/stylesheets/colours/_palette.scss#L50

(I like that the button variable has been extracted out as a concept)